### DISCUSSION TITLE
Does it provide a variable to say whether it's on mobile(phone) or PC ？

### DISCUSSION THREAD

**Discussion by Chasen-Zhang** [facebook/docusaurus#5858] [General] [answered]
Does it provide a variable to say whether it's on mobile(phone) or PC ？

**Comment by Josh-Cena** [comment, marked as answer]
There's a `@theme/hooks/useWindowSize` hook which you can use in your React app. `useWindowSize()` returns `desktop`, `mobile`, or `ssr` if it is server-side rendered. We use it ourselves:

https://github.com/facebook/docusaurus/blob/44b87c7db8e3f47445ed3400890e5b8a10f75d74/packages/docusaurus-theme-classic/src/theme/DocSidebar/index.tsx#L123-L136
